### ISSUE TITLE
Could you please provide the code for preprocessing from protein file to tensor?

### ISSUE THREAD

**Issue by z-linlinlin** [fouticus/pipgcn#16]
I want to run some test on my own data, but I don't know how to do the transformation from pdb files to model inputs.

**Comment by fouticus** [comment]
Hi, I'll have to check with my coauthor who wrote most of the feature generation code to see if they are willing to share it. In the meantime you can see the feature descriptions in Appendix A of [here](https://mountainscholar.org/handle/10217/185661).

**Comment by z-linlinlin** [comment]
Okay, thanks !

**Comment by RujYin** [comment]
Yes, the feature generation code would be very helpful for me too!

**Comment by jhwma** [comment]
Hi there, I was also hoping to regenerate some of these features and was wondering if the code for this was ever made available?
